# German translations for sahara.
# Copyright (C) 2014 ORGANIZATION
# This file is distributed under the same license as the sahara project.
#
# Translators:
# Carsten Duch <cad@teuto.net>, 2014
# Robert Simai, 2014
msgid ""
msgstr ""
"Project-Id-Version:  Sahara\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2014-08-04 06:10+0000\n"
"PO-Revision-Date: 2014-07-17 06:41+0000\n"
"Last-Translator: openstackjenkins <jenkins@openstack.org>\n"
"Language-Team: German "
"(http://www.transifex.com/projects/p/sahara/language/de/)\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 1.3\n"

#: sahara/config.py:94
#, python-format
msgid "Option '%(option)s' is required for config group '%(group)s'"
msgstr ""

#: sahara/config.py:102
msgid ""
"use_namespaces can not be set to \"True\" when use_neutron is set to "
"\"False\""
msgstr ""

#: sahara/context.py:109
msgid "Context isn't available here"
msgstr ""

#: sahara/exceptions.py:27
msgid "An unknown exception occurred"
msgstr ""

#: sahara/exceptions.py:39
#, python-format
msgid "Object '%s' is not found"
msgstr ""

#: sahara/exceptions.py:51
msgid "Name already exists"
msgstr ""

#: sahara/exceptions.py:60
msgid "Invalid credentials"
msgstr ""

#: sahara/exceptions.py:69
msgid "Invalid object reference"
msgstr ""

#: sahara/exceptions.py:78
#, python-format
msgid "Error during command execution: \"%s\""
msgstr ""

#: sahara/exceptions.py:109
msgid "Data is invalid"
msgstr ""

#: sahara/exceptions.py:118
msgid "Job binary internal data must be a string of length greater than zero"
msgstr ""

#: sahara/exceptions.py:128
msgid ""
"To work with JobBinary located in internal swift add 'user' and "
"'password' to extra"
msgstr ""

#: sahara/exceptions.py:138
msgid "Database object already exists"
msgstr ""

#: sahara/exceptions.py:147
msgid "Object was not deleted"
msgstr ""

#: sahara/exceptions.py:157
#, python-format
msgid "Node Group %s is missing 'floating_ip_pool' field"
msgstr ""

#: sahara/exceptions.py:175
#, python-format
msgid "Size of data (%(size)s) is greater than maximum (%(maximum)s)"
msgstr ""

#: sahara/exceptions.py:187
#, python-format
msgid "An error occurred in thread '%(thread)s': %(e)s"
msgstr ""

#: sahara/exceptions.py:197
#, python-format
msgid "Feature '%s' is not implemented"
msgstr ""

#: sahara/exceptions.py:203
#, python-format
msgid "Heat stack failed with status %s"
msgstr ""

#: sahara/exceptions.py:237
#, python-format
msgid "Operation timed out after %i second(s)"
msgstr ""

#: sahara/api/base.py:22
msgid "This API operation isn't implemented"
msgstr ""

#: sahara/conductor/__init__.py:32
msgid "Remote conductor isn't implemented yet."
msgstr ""

#: sahara/conductor/resource.py:118
#, python-format
msgid "Unsupported type: %s"
msgstr ""

#: sahara/db/migration/cli.py:41
msgid "You must provide a revision or relative delta"
msgstr ""

#: sahara/db/sqlalchemy/api.py:192
#, python-format
msgid "Duplicate entry for Cluster: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:202
#, python-format
msgid "Duplicate entry for NodeGroup: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:214 sahara/db/sqlalchemy/api.py:226
#: sahara/db/sqlalchemy/api.py:245
#, python-format
msgid "Cluster id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:261 sahara/db/sqlalchemy/api.py:273
#: sahara/db/sqlalchemy/api.py:292
#, python-format
msgid "Node Group id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:311 sahara/db/sqlalchemy/api.py:322
#: sahara/db/sqlalchemy/api.py:339 sahara/db/sqlalchemy/api.py:350
#, python-format
msgid "Instance id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:383
#, python-format
msgid "Duplicate entry for ClusterTemplate: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:394
#, python-format
msgid "Duplicate entry for TemplatesRelation: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:407
#, python-format
msgid "Cluster Template id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:439
#, python-format
msgid "Duplicate entry for NodeGroupTemplate: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:452
#, python-format
msgid "Node Group Template id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:483
#, python-format
msgid "Duplicate entry for DataSource: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:496
#, python-format
msgid "Data Source id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:500 sahara/db/sqlalchemy/api.py:637
msgid " on foreign key constraint"
msgstr ""

#: sahara/db/sqlalchemy/api.py:501
#, python-format
msgid "Data Source deletion failed%s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:535
#, python-format
msgid "Duplicate entry for JobExecution: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:547 sahara/db/sqlalchemy/api.py:559
#, python-format
msgid "JobExecution id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:608
#, python-format
msgid "Duplicate entry for Job: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:620 sahara/db/sqlalchemy/api.py:633
#, python-format
msgid "Job id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:638
#, python-format
msgid "Job deletion failed%s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:679
#, python-format
msgid "Duplicate entry for JobBinary: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:700
#, python-format
msgid "JobBinary id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:704
msgid "JobBinary is referenced and cannot be deleted"
msgstr ""

#: sahara/db/sqlalchemy/api.py:744 sahara/db/sqlalchemy/api.py:762
#, python-format
msgid ""
"Size of internal binary (%(size)sKB) is greater than the maximum "
"(%(maximum)sKB)"
msgstr ""

#: sahara/db/sqlalchemy/api.py:774
#, python-format
msgid "Duplicate entry for JobBinaryInternal: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:787
#, python-format
msgid "JobBinaryInternal id '%s' not found!"
msgstr ""

#: sahara/middleware/auth_valid.py:55
msgid "Incorrect path"
msgstr ""

#: sahara/middleware/auth_valid.py:61
msgid "Token tenant != requested tenant"
msgstr ""

#: sahara/openstack/common/exception.py:103
msgid "Uncaught exception"
msgstr "Nicht abgefangene Ausnahme"

#: sahara/openstack/common/gettextutils.py:301
msgid "Message objects do not support addition."
msgstr "Das Nachrichtenobjekt unterstützt kein Hinzufügen."

#: sahara/openstack/common/gettextutils.py:311
msgid ""
"Message objects do not support str() because they may contain non-ascii "
"characters. Please use unicode() or translate() instead."
msgstr ""
"Nachrichtenobjekte unterstützen kein str() denn sie können nicht-ASCII "
"Zeichen enthalten. Bitte verwenden Sie statt dessen unicode() oder "
"translate()."

#: sahara/openstack/common/lockutils.py:101
#, python-format
msgid "Unable to acquire lock on `%(filename)s` due to %(exception)s"
msgstr ""

#: sahara/openstack/common/log.py:276
#, python-format
msgid "Deprecated: %s"
msgstr "Veraltet: %s"

#: sahara/openstack/common/log.py:385
#, python-format
msgid "Error loading logging config %(log_config)s: %(err_msg)s"
msgstr "Fehler beim Laden der Logging-Konfiguration %(log_config)s: %(err_msg)s"

#: sahara/openstack/common/log.py:444
#, python-format
msgid "syslog facility must be one of: %s"
msgstr "syslog-Funktion muss einer der folgenden Werte sein: %s"

#: sahara/openstack/common/log.py:687
#, python-format
msgid "Fatal call to deprecated config: %(msg)s"
msgstr "Schwerwiegender Aufruf einer veralteten Konfiguration: %(msg)s"

#: sahara/openstack/common/periodic_task.py:40
#, python-format
msgid "Unexpected argument for periodic task creation: %(arg)s."
msgstr ""
"Unerwartetes Argument zur Erzeugung einer wiederkehrenden Aufgabe: "
"%(arg)s."

#: sahara/openstack/common/processutils.py:59
msgid "Unexpected error while running command."
msgstr "Unerwarteter Fehler beim Kommandoaufruf."

#: sahara/openstack/common/processutils.py:62
#, python-format
msgid ""
"%(description)s\n"
"Command: %(cmd)s\n"
"Exit code: %(exit_code)s\n"
"Stdout: %(stdout)r\n"
"Stderr: %(stderr)r"
msgstr ""
"%(description)s\n"
"Kommando: %(cmd)s\n"
"Rückgabewert: %(exit_code)s\n"
"Stdout: %(stdout)r\n"
"Stderr: %(stderr)r"

#: sahara/openstack/common/processutils.py:143
#, python-format
msgid "Got unknown keyword args: %r"
msgstr ""

#: sahara/openstack/common/processutils.py:148
msgid "Command requested root, but did not specify a root helper."
msgstr ""
"Das Kommando erfordert root-Zugriff, aber es wurde kein root-"
"Hilfsprogramm angegeben."

#: sahara/openstack/common/processutils.py:245
msgid "Environment not supported over SSH"
msgstr "Umgebung über SSH nicht unterstützt"

#: sahara/openstack/common/processutils.py:249
msgid "process_input not supported over SSH"
msgstr "'process_input' über SSH nicht unterstützt"

#: sahara/openstack/common/strutils.py:114
#, python-format
msgid "Unrecognized value '%(val)s', acceptable values are: %(acceptable)s"
msgstr "Unbekannter Wert '%(val)s', bekannte Werte sind: %(acceptable)s"

#: sahara/openstack/common/strutils.py:219
#, python-format
msgid "Invalid unit system: \"%s\""
msgstr "Ungültiges Einheitensystem: \"%s\""

#: sahara/openstack/common/strutils.py:228
#, python-format
msgid "Invalid string format: %s"
msgstr "Ungültiges String Format: %s"

#: sahara/openstack/common/wsgi.py:187
msgid "cannot understand JSON"
msgstr "verstehe JSON nicht"

#: sahara/openstack/common/wsgi.py:211
msgid "cannot understand XML"
msgstr "Verstehe XML nicht"

#: sahara/service/direct_engine.py:286
#, python-format
msgid "Node %s has error status"
msgstr ""

#: sahara/service/validation.py:77
#, python-format
msgid "Object with %s not found"
msgstr ""

#: sahara/service/volumes.py:62
#, python-format
msgid "Error attach volume to instance %s"
msgstr ""

#: sahara/service/volumes.py:91
#, python-format
msgid "Volume %s has error status"
msgstr ""

#: sahara/service/edp/job_manager.py:104
#, python-format
msgid "Cluster does not support job type %s"
msgstr ""

#: sahara/service/edp/binary_retrievers/internal_swift.py:56
#, python-format
msgid "Url for binary in internal swift must start with %s"
msgstr ""

#: sahara/service/edp/binary_retrievers/internal_swift.py:74
#, python-format
msgid ""
"Size of swift container (%(size)sKB) is greater than maximum "
"(%(maximum)sKB)"
msgstr ""

#: sahara/service/edp/binary_retrievers/internal_swift.py:98
#, python-format
msgid "Size of swift object (%(size)sKB) is greater than maximum (%(maximum)sKB)"
msgstr ""

#: sahara/service/edp/oozie/workflow_creator/base_workflow.py:49
#, python-format
msgid "\"%s\" child cannot be added to prepare element"
msgstr ""

#: sahara/service/edp/oozie/workflow_creator/base_workflow.py:60
#, python-format
msgid "\"%s\" child cannot be added to streaming element"
msgstr ""

#: sahara/service/validations/base.py:53
#, python-format
msgid "Sahara doesn't contain plugin with name '%s'"
msgstr ""

#: sahara/service/validations/base.py:59
#, python-format
msgid "Requested plugin '%(name)s' doesn't support version '%(version)s'"
msgstr ""

#: sahara/service/validations/base.py:66
#, python-format
msgid "Requested image '%s' is not registered"
msgstr ""

#: sahara/service/validations/base.py:77
#, python-format
msgid "Plugin doesn't contain applicable target '%s'"
msgstr ""

#: sahara/service/validations/base.py:82
#, python-format
msgid ""
"Plugin's applicable target '%(target)s' doesn't contain config with name "
"'%(name)s'"
msgstr ""

#: sahara/service/validations/base.py:139
#, python-format
msgid "Requested flavor '%s' not found"
msgstr ""

#: sahara/service/validations/base.py:154
#, python-format
msgid "Floating IP pool %(pool)s for node group '%(group)s' not found"
msgstr ""

#: sahara/service/validations/base.py:161
msgid "Duplicates in node processes have been detected"
msgstr ""

#: sahara/service/validations/base.py:169
#, python-format
msgid "Plugin supports the following node procesess: %s"
msgstr ""

#: sahara/service/validations/base.py:177
msgid "Duplicates in node group names are detected"
msgstr ""

#: sahara/service/validations/base.py:185
#, python-format
msgid "Cluster with name '%s' already exists"
msgstr ""

#: sahara/service/validations/base.py:194
#, python-format
msgid "Cluster name '%s' is already used as Heat stack name"
msgstr ""

#: sahara/service/validations/base.py:201
#, python-format
msgid "Cluster with id '%s' doesn't exist"
msgstr ""

#: sahara/service/validations/base.py:212
#, python-format
msgid ""
"Composite hostname %(host)s in provisioned cluster exceeds maximum limit "
"%(limit)s characters"
msgstr ""

#: sahara/service/validations/base.py:223
#, python-format
msgid "Requested keypair '%s' not found"
msgstr ""

#: sahara/service/validations/base.py:228
#, python-format
msgid "Network %s not found"
msgstr ""

#: sahara/service/validations/base.py:236
#, python-format
msgid "Cluster template with name '%s' already exists"
msgstr ""

#: sahara/service/validations/base.py:242
#, python-format
msgid "Cluster template with id '%s' doesn't exist"
msgstr ""

#: sahara/service/validations/base.py:262
#, python-format
msgid "NodeGroup template with name '%s' already exists"
msgstr ""

#: sahara/service/validations/base.py:268
#, python-format
msgid "NodeGroup template with id '%s' doesn't exist"
msgstr ""

#: sahara/service/validations/base.py:303
#, python-format
msgid "Cluster doesn't contain node group with name '%s'"
msgstr ""

#: sahara/service/validations/base.py:317
#, python-format
msgid "Can't add new nodegroup. Cluster already has nodegroup with name '%s'"
msgstr ""

#: sahara/service/validations/base.py:330
msgid "Cinder is not supported"
msgstr ""

#: sahara/service/validations/base.py:342
#, python-format
msgid ""
"Tags of requested image '%(image)s' don't contain required tags "
"['%(name)s', '%(version)s']"
msgstr ""

#: sahara/service/validations/cluster_templates.py:137
#, python-format
msgid "Cluster template %(id)s in use by %(clusters)s"
msgstr ""

#: sahara/service/validations/clusters.py:75
#: sahara/service/validations/clusters.py:97
#, python-format
msgid "'%s' field is not found"
msgstr ""

#: sahara/service/validations/clusters.py:91
msgid "'neutron_management_network' field can't be used with 'use_neutron=False'"
msgstr ""

#: sahara/service/validations/clusters_scaling.py:77
#, python-format
msgid "Requested plugin '%s' doesn't support cluster scaling feature"
msgstr ""

#: sahara/service/validations/clusters_scaling.py:82
#, python-format
msgid "Cluster cannot be scaled not in 'Active' status. Cluster status: %s"
msgstr ""

#: sahara/service/validations/node_group_templates.py:111
#, python-format
msgid ""
"Node group template %(template)s is in use by cluster templates: "
"%(users)s; and clusters: %(clusters)s"
msgstr ""

#: sahara/service/validations/plugins.py:27
#, python-format
msgid ""
"Requested plugin '%s' doesn't support converting config files to cluster "
"templates"
msgstr ""

#: sahara/topology/topology_helper.py:113
#, python-format
msgid "Unable to find file %s with compute topology"
msgstr ""

#: sahara/topology/topology_helper.py:129
#, python-format
msgid "Was not able to find compute node topology for VM %s"
msgstr ""

#: sahara/utils/api.py:137
msgid "Non-dict and non-empty kwargs passed to render"
msgstr ""

#: sahara/utils/api.py:159 sahara/utils/api.py:188
#, python-format
msgid "Content type '%s' isn't supported"
msgstr ""

#: sahara/utils/api.py:184
msgid "XML requests are not supported yet"
msgstr ""

#: sahara/utils/crypto.py:51
msgid "Private key file hasn't been created"
msgstr ""

#: sahara/utils/crypto.py:55
msgid "Public key file hasn't been created"
msgstr ""

#: sahara/utils/remote.py:123
msgid ""
"Remote driver is not loaded. Most probably you see this error because you"
" are running Sahara in distributed mode and it is broken.Try running "
"sahara-all instead."
msgstr ""

#: sahara/utils/tempfiles.py:37
#, python-format
msgid "Failed to delete temp dir %(dir)s (reason: %(reason)s)"
msgstr ""

#: sahara/utils/types.py:88
#, python-format
msgid "Class %s is immutable!"
msgstr ""

#: sahara/utils/openstack/base.py:45
#, python-format
msgid "Endpoint with type %(type)s is not found for service %(service)s"
msgstr ""

#: sahara/utils/openstack/base.py:52
#, python-format
msgid "Service \"%s\" not found in service catalog"
msgstr ""

#: sahara/utils/openstack/heat.py:45
#, python-format
msgid "Failed to find stack %(stack)s"
msgstr ""

#: sahara/utils/openstack/neutron.py:78
#, python-format
msgid "Neutron router corresponding to network %s is not found"
msgstr ""

#: sahara/utils/openstack/neutron.py:185
#, python-format
msgid "Unknown file mode %s"
msgstr ""

